wsmux/post-test-data

```diff
@@ -0,0 +1,2 @@
+This string is supposed to be sent over 
```

Comment:
@ckousik, you can do that with a custom stream too... Just wrap the stream:
```go
type wrapStream struct {
  *bytes.Buffer
}
```

And it'll all work fine without the need for a file stream.. the only reason `bytes.Buffer` isn't using chunked encoding is that `http.Request` module knows how cast to that type... it can't do that with a custom type.